src/coreComponents/physicsSolvers/multiphysics/integratedTests/Hydrofracture_KGD_EdgeBased_C3D6.xml

```diff
@@ -34,7 +34,7 @@
         newtonTol="1.0e-6"
         newtonMaxIter="5"/>
       <LinearSolverParameters
-        solverType="gmres"
+        solverType="direct"
         krylovTol="1.0e-10"
```

Comment:
A fully implicit coupling option is used. These lines can be safely removed:

https://github.com/GEOSX/GEOSX/blob/780dfba0943bfe612c0ad82a6c8f230c340be47e/src/coreComponents/physicsSolvers/multiphysics/integratedTests/Hydrofracture_KGD_EdgeBased_C3D6.xml#L33-L39